docs/sections/user_guide/yaml/execution.rst

```diff
@@ -85,4 +86,4 @@ The MPI launch program (``mpiexec``, ``srun``, et al.). This entry is only used
 threads:
 """"""""
 
-Used to set the ``OMP_NUM_THREADS`` environment variable in the execution environment, to control the number of OpenMP threads to use when running the component. This entry is only used when configuring parallel components.
+For drivers implementing support, sets the ``OMP_NUM_THREADS`` environment variable in the execution environment, to control the number of OpenMP threads to use when running the component. This entry is only used when configuring parallel components.
```

Comment:
It's necessary for each driver that supports OpenMP configuration via the `OMP_NUM_THREADS` environment variable to support it in its `runscript()` method. Each driver needs to know whether its component supports OpenMP threading.